src/components/MultiLink/Legend.vue

```diff
@@ -226,7 +226,12 @@ export default Vue.extend({
       });
     },
 
+    // TODO: #176 use table name for var selection
     isQuantitative(varName: string, type: 'node' | 'link') {
+      if (Object.entries(this.columnTypes).length > 0) {
```

Comment:
Idiomatic usage would be to use `Object.keys()` here instead of `Object.entries()`.

This whole function is pretty odd--could you explain what it's doing? The heuristic "guess" at whether a variable is numeric after this block you've added makes me think there ought to be a Multinet-native way to do this.

Reply:
this function just if the multinet type of the variable is 'number' and returns true if it is. 

The heuristic is just for the data that is not typed in the database. Ultimately, we will probably want types on all data and the heuristic could be removed.

Reply:
See 7495a34 for `.entries()` -> `.keys()`

Reply:
I'm going to go ahead and merge. Let's open an issue if you think this needs work